### DISCUSSION TITLE
Offline installation

### DISCUSSION THREAD

**Discussion by bastionji** [frangoteam/FUXA#758] [Q&A]
Hi, how can I install Fuxa in offline machines?
I mean, in a server not connected to internet but only in a field network.

Thanks in advance.

**Comment by unocelli** [comment]
Hi, we have never tried, maybe with `npm pack`

**Comment by voskuil23** [comment]
I have the same situation, where I have an internal field network of devices which is isolated from the internet.

Could you please help with a possible solution?

Cheers :)

**Comment by robbudge** [comment]
I think you would need internet access to download the specific packages for the hardware during the build process. 
I suspect if you performed this once online and looking at the required packages it’s probably possible to have them stored locally for install. 

We always build on-line then the system goes off-line for security. 

You would need to look into manual install / builds with NPM.
